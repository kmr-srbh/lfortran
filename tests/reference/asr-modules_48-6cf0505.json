{
    "basename": "asr-modules_48-6cf0505",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/modules_48.f90",
    "infile_hash": "83255edc16b2873de5a0b7431a7679c296d40c86424c2a7f1ca81fff",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-modules_48-6cf0505.stdout",
    "stdout_hash": "fc9f30a7e0ed02cacbeb25ef5516ce0f05baa312b25a9a7d592ccb29",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}