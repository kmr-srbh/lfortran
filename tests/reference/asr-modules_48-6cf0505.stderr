semantic error: The RHS character len must be known at compile time
  --> tests/../integration_tests/modules_48.f90:52:5
   |
52 |     character(len=*), parameter :: int = '0123456789'
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
